M2/Macaulay2/d/actors4.d

```diff
@@ -1586,123 +1587,30 @@ precision(e:Expr):Expr := (
 setupfun("precision0",precision);
 
 -- locate:
+locate(p:Position):Expr := (
```

Comment:
Same here, could you move this to `debugging.dd` (or maybe `code.dd`)? Aligning the names with the filename in Core would be ideal, but at the very least not in a jungle of actors is great.